packages/usa-breadcrumb/src/styles/_usa-breadcrumb.scss

```diff
@@ -87,7 +87,6 @@ $breadcrumb-back-icon-aspect: (
 .usa-breadcrumb__list-item {
   @include sr-only;
   @include u-display("inline");
-  @include u-white-space("no-wrap");
```

Comment:
This appears to be redundant because `no-wrap` is already declared on the `.usa-breadcrumb__list` element in [line 71](https://github.com/uswds/uswds/pull/5497/files#diff-75c264f6f911cdcf26091f2e3482b2db353104d08cc070516ac20ece73b99255L71)